authz-js-policies/README.md

```diff
@@ -0,0 +1,84 @@
+authz-js-policies: Example of JavaScript authorization policy deployable to Keycloak server
+===========================================================================================
+
+Level: Intermediate
+Technologies: JavaEE
+Summary: Example of JavaScript authorization policy
+Target Product: Keycloak  
+Source: <https://github.com/keycloak/keycloak-quickstarts>  
+
+
+What is it?
+-----------
+
+TODO
+
+System Requirements
+-------------------
+
+You need to have Keycloak 18.0.Final running.
```

Comment:
perhaps better "at least Keycloak 18.0.Final" or sth? I'd avoid strict version pinning in a general example.

Reply:
Yes, true. Thanks for these points. I've copy/pasted this from the README of some other example :-) Similarly the Java version.

It seems that README files of the quickstarts, which are deployable to Keycloak server, may need some revision as they are currently target just for the Keycloak on Wildfly.

This particular quickstart didn't have README file like the other quickstarts have. So I've added README file, but for this README, I've added lots of TODO and I focused only on parts specific to instructions for deployment on Keycloak on Wildfly VS Keycloak on Quarkus as this is the most important part for the particular use-case addressed here for the scripts providers. I can properly update the README file and remove all TODOs if we agree that my approach for script providers work.